ocs_ci/ocs/resources/pod.py

```diff
@@ -2946,3 +2946,52 @@ def restart_pods_in_statuses(
     )
     delete_pods(pods_to_restart, wait=wait)
     logger.info("Finish restarting the pods")
+
+
+def wait_for_ceph_cmd_execute_successfully(timeout=300):
+    """
+    Wait for a Ceph command to execute successfully
+
+    Args:
+        timeout (int): The time to wait for a Ceph command to execute successfully
+
+    Returns:
+        bool: True, if the Ceph command executed successfully. False, otherwise
+
+    """
+    try:
+        for res in TimeoutSampler(
+            timeout=timeout, sleep=10, func=check_ceph_cmd_execute_successfully
+        ):
+            if res:
+                return True
+    except TimeoutExpiredError:
+        logger.warning(f"Failed to execute the ceph command after {timeout} seconds")
+        return False
+
+
+def check_ceph_cmd_execute_successfully():
+    """
+    Check that a Ceph command executes successfully
+
+    Returns:
+        bool: True, if the Ceph command executed successfully. False, otherwise
+
+    """
+    # Import here to avoid circular loop
```

Comment:
Why does having those modules imported properly at the beginning of the file cause circular import?

Reply:
These are the errors I get when putting them at the beginning of the file:

> File "/home/ikave/RedhatProjects/ocs-ci/ocs_ci/framework/pytest_customization/ocscilib.py", line 31, in <module>
    from ocs_ci.ocs.cluster import check_clusters
  File "/home/ikave/RedhatProjects/ocs-ci/ocs_ci/ocs/cluster.py", line 22, in <module>
    import ocs_ci.ocs.resources.pod as pod
  File "/home/ikave/RedhatProjects/ocs-ci/ocs_ci/ocs/resources/pod.py", line 18, in <module>
    from ocs_ci.ocs.cluster import is_ms_consumer_cluster
ImportError: Error importing plugin "ocs_ci.framework.pytest_customization.ocscilib": cannot import name 'is_ms_consumer_cluster' from partially initialized module 'ocs_ci.ocs.cluster' (most likely due to a circular import) (/home/ikave/RedhatProjects/ocs-ci/ocs_ci/ocs/cluster.py)

> File "/home/ikave/RedhatProjects/ocs-ci/ocs_ci/framework/pytest_customization/ocscilib.py", line 31, in <module>
    from ocs_ci.ocs.cluster import check_clusters
  File "/home/ikave/RedhatProjects/ocs-ci/ocs_ci/ocs/cluster.py", line 22, in <module>
    import ocs_ci.ocs.resources.pod as pod
  File "/home/ikave/RedhatProjects/ocs-ci/ocs_ci/ocs/resources/pod.py", line 18, in <module>
    from ocs_ci.ocs.managedservice import patch_consumer_toolbox
  File "/home/ikave/RedhatProjects/ocs-ci/ocs_ci/ocs/managedservice.py", line 11, in <module>
    from ocs_ci.ocs.resources.pod import get_ceph_tools_pod, get_pods_having_label, Pod
ImportError: Error importing plugin "ocs_ci.framework.pytest_customization.ocscilib": cannot import name 'get_ceph_tools_pod' from partially initialized module 'ocs_ci.ocs.resources.pod' (most likely due to a circular import) (/home/ikave/RedhatProjects/ocs-ci/ocs_ci/ocs/resources/pod.py)